ios/Mobile.framework/Versions/A/Headers/Mobile.objc.h

```diff
@@ -28,9 +28,6 @@
 - (void)notify:(MobileEvent*)event;
 @end
 
-/**
- * Device is a simple meta data wrapper around a Device
- */
```

Comment:
For some reason, whenever you or Carson build the framework, these comments disappear. Weird.